cards/src/main/resources/cards/custom/group0/other_death_knight/tgt/spell_chain_hook.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Chain Hook",
+  "baseManaCost": 7,
+  "type": "SPELL",
+  "heroClass": "SPIRIT",
+  "rarity": "RARE",
+  "description": "Put a minion into your hand.",
```

Comment:
`Return a minion to your hand.` Does this work exactly like Sap, but opponent's minions? Use the same text.